playground/src/state.tsx

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import { useEffect, useId } from "preact/hooks";
```

Comment:
If it's just to fetch the element in the useEffect hook to update it, then should probably use `useRef` here like we do in other components, not use `useId` then querySelector on the id.

Reply:
Changed code to useRef.